lib/devise/multi_email/models/validatable.rb

```diff
@@ -7,8 +7,13 @@ module EmailValidatable
 
       included do
         validates_presence_of   :email, if: :email_required?
-        validates_uniqueness_of :email, allow_blank: true, if: :email_changed?
-        validates_format_of     :email, with: email_regexp, allow_blank: true, if: :email_changed?
+        if Devise.activerecord51?
+          validates_uniqueness_of :email, allow_blank: true, case_sensitive: true, if: :will_save_change_to_email?
+          validates_format_of     :email, with: email_regexp, allow_blank: true, if: :will_save_change_to_email?
+        else
+          validates_uniqueness_of :email, allow_blank: true, if: :email_changed?
```

Comment:
Sorry I haven't worked on Rails for a while, can you tell me why `case_sensitive` is removed? Did rails6 make this the default?

Reply:
Also why `will_save_change_to_email` is changed to `email_changed?`, some comments might be helpful.

Reply:
It appears that `case_sensitive` is being deprecated and will be removed in rails 6.1. As for switching `will_save_change_to_email` moving to `email_changed` ⎯ I'm not really sure. When I went looking for an answer, I found this code in sister code [devise issue 5064](https://github.com/plataformatec/devise/issues/5064) and figured that if it's the direction they are taking, it's probably representative of best practices for us.